### PR TITLE
Update translations and community information

### PR DESCRIPTION
**Pull request by jinhojang6**
- Translation
   - Add Filipino and Indonesian translations
   - Add i18n keys for the nightly page
   - Removed outdated i18n key

- Community information
  - Add Filipino community information

### CODE REVIEW

File: themes/navy/layout/get-involved.ejs [hunk 1]
```diff
@@ -332,6 +332,19 @@
 								</ul>    
 							</div>    
 						</li>
+						<li class="contributor col-md-1">       
+							<a href="#" class="contributor-trigger">
+								<%- image_tag("/img/nicolas.png", { srcset: '/img/nicolas.png 2x'}) %>
```
Comment (jakubgs) [inline themes/navy/layout/get-involved.ejs:337]: You shouldn't mix unrelated changes in one commit. This is not in any way related to translation changes so should be it's own commit.

**Comment by jakubgs** [comment]
Btw, you are aware that the `dev-lang` branch will be deleted after this PR is merged, right?

File: themes/navy/layout/get-involved.ejs (hunk 1 above)
Comment (jinhojang6) [reply themes/navy/layout/get-involved.ejs:337]: Thanks for the feedback, let me split commits if there is something unrelated from next PR on.

**Comment by jinhojang6** [comment]
@jakubgs Yes, I'm aware of it. Let me make a `dev-lang` branch once again when I get to have translation updates in the future
